{
  "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785",
  "repository_url": "https://api.github.com/repos/ethereum/go-ethereum",
  "labels_url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785/labels{/name}",
  "comments_url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785/comments",
  "events_url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785/events",
  "html_url": "https://github.com/ethereum/go-ethereum/issues/1785",
  "id": 105636618,
  "node_id": "MDU6SXNzdWUxMDU2MzY2MTg=",
  "number": 1785,
  "title": "accounts: unlock multiple accounts fails",
  "user": {
    "login": "danielnovy",
    "id": 3409753,
    "node_id": "MDQ6VXNlcjM0MDk3NTM=",
    "avatar_url": "https://avatars.githubusercontent.com/u/3409753?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/danielnovy",
    "html_url": "https://github.com/danielnovy",
    "followers_url": "https://api.github.com/users/danielnovy/followers",
    "following_url": "https://api.github.com/users/danielnovy/following{/other_user}",
    "gists_url": "https://api.github.com/users/danielnovy/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/danielnovy/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/danielnovy/subscriptions",
    "organizations_url": "https://api.github.com/users/danielnovy/orgs",
    "repos_url": "https://api.github.com/users/danielnovy/repos",
    "events_url": "https://api.github.com/users/danielnovy/events{/privacy}",
    "received_events_url": "https://api.github.com/users/danielnovy/received_events",
    "type": "User",
    "site_admin": false
  },
  "labels": [
    {
      "id": 72233650,
      "node_id": "MDU6TGFiZWw3MjIzMzY1MA==",
      "url": "https://api.github.com/repos/ethereum/go-ethereum/labels/type:bug",
      "name": "type:bug",
      "color": "FF5E5E",
      "default": false,
      "description": ""
    }
  ],
  "state": "closed",
  "locked": false,
  "assignee": {
    "login": "fjl",
    "id": 6915,
    "node_id": "MDQ6VXNlcjY5MTU=",
    "avatar_url": "https://avatars.githubusercontent.com/u/6915?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/fjl",
    "html_url": "https://github.com/fjl",
    "followers_url": "https://api.github.com/users/fjl/followers",
    "following_url": "https://api.github.com/users/fjl/following{/other_user}",
    "gists_url": "https://api.github.com/users/fjl/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/fjl/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/fjl/subscriptions",
    "organizations_url": "https://api.github.com/users/fjl/orgs",
    "repos_url": "https://api.github.com/users/fjl/repos",
    "events_url": "https://api.github.com/users/fjl/events{/privacy}",
    "received_events_url": "https://api.github.com/users/fjl/received_events",
    "type": "User",
    "site_admin": false
  },
  "assignees": [
    {
      "login": "fjl",
      "id": 6915,
      "node_id": "MDQ6VXNlcjY5MTU=",
      "avatar_url": "https://avatars.githubusercontent.com/u/6915?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/fjl",
      "html_url": "https://github.com/fjl",
      "followers_url": "https://api.github.com/users/fjl/followers",
      "following_url": "https://api.github.com/users/fjl/following{/other_user}",
      "gists_url": "https://api.github.com/users/fjl/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/fjl/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/fjl/subscriptions",
      "organizations_url": "https://api.github.com/users/fjl/orgs",
      "repos_url": "https://api.github.com/users/fjl/repos",
      "events_url": "https://api.github.com/users/fjl/events{/privacy}",
      "received_events_url": "https://api.github.com/users/fjl/received_events",
      "type": "User",
      "site_admin": false
    }
  ],
  "milestone": {
    "url": "https://api.github.com/repos/ethereum/go-ethereum/milestones/20",
    "html_url": "https://github.com/ethereum/go-ethereum/milestone/20",
    "labels_url": "https://api.github.com/repos/ethereum/go-ethereum/milestones/20/labels",
    "id": 1299388,
    "node_id": "MDk6TWlsZXN0b25lMTI5OTM4OA==",
    "number": 20,
    "title": "1.4.0",
    "description": "",
    "creator": {
      "login": "obscuren",
      "id": 6264126,
      "node_id": "MDQ6VXNlcjYyNjQxMjY=",
      "avatar_url": "https://avatars.githubusercontent.com/u/6264126?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/obscuren",
      "html_url": "https://github.com/obscuren",
      "followers_url": "https://api.github.com/users/obscuren/followers",
      "following_url": "https://api.github.com/users/obscuren/following{/other_user}",
      "gists_url": "https://api.github.com/users/obscuren/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/obscuren/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/obscuren/subscriptions",
      "organizations_url": "https://api.github.com/users/obscuren/orgs",
      "repos_url": "https://api.github.com/users/obscuren/repos",
      "events_url": "https://api.github.com/users/obscuren/events{/privacy}",
      "received_events_url": "https://api.github.com/users/obscuren/received_events",
      "type": "User",
      "site_admin": false
    },
    "open_issues": 0,
    "closed_issues": 141,
    "state": "closed",
    "created_at": "2015-09-10T13:53:50Z",
    "updated_at": "2016-08-17T20:07:24Z",
    "due_on": null,
    "closed_at": "2016-05-10T10:44:41Z"
  },
  "comments": 6,
  "created_at": "2015-09-09T16:43:05Z",
  "updated_at": "2016-05-10T09:57:49Z",
  "closed_at": "2016-05-10T09:57:46Z",
  "author_association": "NONE",
  "active_lock_reason": null,
  "body": "When trying to use `--unlock=\"0 1\"` geth fails with the message: `Fatal: Unlock account failed 'Decryption failed: MAC mismatch'`. The first account is unlocked and the message `Account '0' unlocked` is printed. \n\nIt's important to note that `unlock=\"0\"` AND `unlock=\"1\"` both works! Also, if you invert the order (\"1 0\" instead of \"0 1\"), it prints `Account '1' unlocked` and then fails to unlock second account with same message. So, the problem is clearly related to unlock multiple accounts and problem always happens on second account.\n",
  "closed_by": {
    "login": "obscuren",
    "id": 6264126,
    "node_id": "MDQ6VXNlcjYyNjQxMjY=",
    "avatar_url": "https://avatars.githubusercontent.com/u/6264126?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/obscuren",
    "html_url": "https://github.com/obscuren",
    "followers_url": "https://api.github.com/users/obscuren/followers",
    "following_url": "https://api.github.com/users/obscuren/following{/other_user}",
    "gists_url": "https://api.github.com/users/obscuren/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/obscuren/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/obscuren/subscriptions",
    "organizations_url": "https://api.github.com/users/obscuren/orgs",
    "repos_url": "https://api.github.com/users/obscuren/repos",
    "events_url": "https://api.github.com/users/obscuren/events{/privacy}",
    "received_events_url": "https://api.github.com/users/obscuren/received_events",
    "type": "User",
    "site_admin": false
  },
  "reactions": {
    "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785/reactions",
    "total_count": 0,
    "+1": 0,
    "-1": 0,
    "laugh": 0,
    "hooray": 0,
    "confused": 0,
    "heart": 0,
    "rocket": 0,
    "eyes": 0
  },
  "timeline_url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785/timeline",
  "performed_via_github_app": null,
  "state_reason": "completed"
}
[
  {
    "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/153712851",
    "html_url": "https://github.com/ethereum/go-ethereum/issues/1785#issuecomment-153712851",
    "issue_url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785",
    "id": 153712851,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE1MzcxMjg1MQ==",
    "user": {
      "login": "zelig",
      "id": 769725,
      "node_id": "MDQ6VXNlcjc2OTcyNQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/769725?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/zelig",
      "html_url": "https://github.com/zelig",
      "followers_url": "https://api.github.com/users/zelig/followers",
      "following_url": "https://api.github.com/users/zelig/following{/other_user}",
      "gists_url": "https://api.github.com/users/zelig/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/zelig/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/zelig/subscriptions",
      "organizations_url": "https://api.github.com/users/zelig/orgs",
      "repos_url": "https://api.github.com/users/zelig/repos",
      "events_url": "https://api.github.com/users/zelig/events{/privacy}",
      "received_events_url": "https://api.github.com/users/zelig/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2015-11-04T12:56:12Z",
    "updated_at": "2015-11-04T12:56:12Z",
    "author_association": "MEMBER",
    "body": "I believe this has already been fixed in #1840 \n",
    "reactions": {
      "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/153712851/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/159868011",
    "html_url": "https://github.com/ethereum/go-ethereum/issues/1785#issuecomment-159868011",
    "issue_url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785",
    "id": 159868011,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE1OTg2ODAxMQ==",
    "user": {
      "login": "karalabe",
      "id": 129561,
      "node_id": "MDQ6VXNlcjEyOTU2MQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/129561?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/karalabe",
      "html_url": "https://github.com/karalabe",
      "followers_url": "https://api.github.com/users/karalabe/followers",
      "following_url": "https://api.github.com/users/karalabe/following{/other_user}",
      "gists_url": "https://api.github.com/users/karalabe/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/karalabe/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/karalabe/subscriptions",
      "organizations_url": "https://api.github.com/users/karalabe/orgs",
      "repos_url": "https://api.github.com/users/karalabe/repos",
      "events_url": "https://api.github.com/users/karalabe/events{/privacy}",
      "received_events_url": "https://api.github.com/users/karalabe/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2015-11-26T10:15:42Z",
    "updated_at": "2015-11-26T10:15:42Z",
    "author_association": "MEMBER",
    "body": "Closing if supposedly fixed. Please reopen if it's not the case. Also note, an upcoming PR will change the account list to comma separated values instead of space separated ones. This is a general move towards eliminating whitespace from within CLI arguments as they cause significant pains when passing through different environments (e.g. bash -> go -> docker -> bash).\n",
    "reactions": {
      "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/159868011/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/159899989",
    "html_url": "https://github.com/ethereum/go-ethereum/issues/1785#issuecomment-159899989",
    "issue_url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785",
    "id": 159899989,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE1OTg5OTk4OQ==",
    "user": {
      "login": "danielnovy",
      "id": 3409753,
      "node_id": "MDQ6VXNlcjM0MDk3NTM=",
      "avatar_url": "https://avatars.githubusercontent.com/u/3409753?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/danielnovy",
      "html_url": "https://github.com/danielnovy",
      "followers_url": "https://api.github.com/users/danielnovy/followers",
      "following_url": "https://api.github.com/users/danielnovy/following{/other_user}",
      "gists_url": "https://api.github.com/users/danielnovy/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/danielnovy/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/danielnovy/subscriptions",
      "organizations_url": "https://api.github.com/users/danielnovy/orgs",
      "repos_url": "https://api.github.com/users/danielnovy/repos",
      "events_url": "https://api.github.com/users/danielnovy/events{/privacy}",
      "received_events_url": "https://api.github.com/users/danielnovy/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2015-11-26T12:23:48Z",
    "updated_at": "2015-11-26T12:23:48Z",
    "author_association": "NONE",
    "body": "Sorry for my late reply! Just tried it now and it's still not working in version `1.4.0-unstable`. So, please, reopen the issue since I don't have the privileges to do so. Thanks!\n",
    "reactions": {
      "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/159899989/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/159901966",
    "html_url": "https://github.com/ethereum/go-ethereum/issues/1785#issuecomment-159901966",
    "issue_url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785",
    "id": 159901966,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE1OTkwMTk2Ng==",
    "user": {
      "login": "karalabe",
      "id": 129561,
      "node_id": "MDQ6VXNlcjEyOTU2MQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/129561?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/karalabe",
      "html_url": "https://github.com/karalabe",
      "followers_url": "https://api.github.com/users/karalabe/followers",
      "following_url": "https://api.github.com/users/karalabe/following{/other_user}",
      "gists_url": "https://api.github.com/users/karalabe/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/karalabe/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/karalabe/subscriptions",
      "organizations_url": "https://api.github.com/users/karalabe/orgs",
      "repos_url": "https://api.github.com/users/karalabe/repos",
      "events_url": "https://api.github.com/users/karalabe/events{/privacy}",
      "received_events_url": "https://api.github.com/users/karalabe/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2015-11-26T12:36:21Z",
    "updated_at": "2015-11-26T12:36:21Z",
    "author_association": "MEMBER",
    "body": "I'll take a look\n",
    "reactions": {
      "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/159901966/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/198055623",
    "html_url": "https://github.com/ethereum/go-ethereum/issues/1785#issuecomment-198055623",
    "issue_url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785",
    "id": 198055623,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE5ODA1NTYyMw==",
    "user": {
      "login": "javgh",
      "id": 747896,
      "node_id": "MDQ6VXNlcjc0Nzg5Ng==",
      "avatar_url": "https://avatars.githubusercontent.com/u/747896?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/javgh",
      "html_url": "https://github.com/javgh",
      "followers_url": "https://api.github.com/users/javgh/followers",
      "following_url": "https://api.github.com/users/javgh/following{/other_user}",
      "gists_url": "https://api.github.com/users/javgh/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/javgh/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/javgh/subscriptions",
      "organizations_url": "https://api.github.com/users/javgh/orgs",
      "repos_url": "https://api.github.com/users/javgh/repos",
      "events_url": "https://api.github.com/users/javgh/events{/privacy}",
      "received_events_url": "https://api.github.com/users/javgh/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-03-17T19:51:51Z",
    "updated_at": "2016-03-17T19:51:51Z",
    "author_association": "NONE",
    "body": "I'm also seeing this with geth 1.3.5 - trying to unlock more than one account via `--unlock \"0 1\"` fails on whatever account is listed second with a `'Decryption failed: MAC mismatch'` error, but only when used in combination with the `--password` option. When entering the passwords manually, both accounts are unlocked successfully. I'm also only seeing this error on one of my machines - another one also has two accounts and the same geth version and there it works. At the moment I don't see what the difference might be.\n",
    "reactions": {
      "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/198055623/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/199582269",
    "html_url": "https://github.com/ethereum/go-ethereum/issues/1785#issuecomment-199582269",
    "issue_url": "https://api.github.com/repos/ethereum/go-ethereum/issues/1785",
    "id": 199582269,
    "node_id": "MDEyOklzc3VlQ29tbWVudDE5OTU4MjI2OQ==",
    "user": {
      "login": "thiagohortal",
      "id": 6624667,
      "node_id": "MDQ6VXNlcjY2MjQ2Njc=",
      "avatar_url": "https://avatars.githubusercontent.com/u/6624667?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/thiagohortal",
      "html_url": "https://github.com/thiagohortal",
      "followers_url": "https://api.github.com/users/thiagohortal/followers",
      "following_url": "https://api.github.com/users/thiagohortal/following{/other_user}",
      "gists_url": "https://api.github.com/users/thiagohortal/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/thiagohortal/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/thiagohortal/subscriptions",
      "organizations_url": "https://api.github.com/users/thiagohortal/orgs",
      "repos_url": "https://api.github.com/users/thiagohortal/repos",
      "events_url": "https://api.github.com/users/thiagohortal/events{/privacy}",
      "received_events_url": "https://api.github.com/users/thiagohortal/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-03-22T01:54:06Z",
    "updated_at": "2016-03-22T01:54:06Z",
    "author_association": "NONE",
    "body": "Hello guys,\nI am also getting this return: 'Decryption failed: MAC mismatch', any solution?\nI am using web3 project.\n\nHelp!\n",
    "reactions": {
      "url": "https://api.github.com/repos/ethereum/go-ethereum/issues/comments/199582269/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  }
]
